src/subdomains/supporting/bank/bank-account/bank-account-admin.controller.ts

```diff
@@ -7,7 +7,7 @@ import { BankAccount } from './bank-account.entity';
 import { BankAccountService } from './bank-account.service';
 import { CreateBankAccountDto } from './dto/create-bank-account.dto';
 
-@ApiTags('BankAccount')
+@ApiTags('Bank Account')
```

Comment:
`Bank-Account` or `Bank account` ?

Reply:
![image](https://github.com/DFXswiss/api/assets/85513542/53e22732-c545-4117-a880-f713de9cd3c7)
ApiTags are shown as titles in the Swagger docu and titles are written in capital letters 😉